Makefile

```diff
@@ -38,7 +38,7 @@ GO_DIR := $(OUTPUT_DIR)/go
 
 # Base image used for all golang containers
 # Uses trusted google-built golang image
-GOLANG_IMAGE_VERSION := 1.21.11
+GOLANG_IMAGE_VERSION := 1.22.5
```

Comment:
wanna bump it in go.mod too or are we just doing the environment first?

Reply:
i guess it's fine to just bump the build containers first.

Reply:
Just the environment first